src/modules/ContentList/index.tsx

```diff
@@ -9,28 +9,38 @@ import { Tabs } from '@aiera/client-sdk/components/Tabs';
 import { getPrimaryQuote, useCompanyResolver } from '@aiera/client-sdk/lib/data';
 import { useChangeHandlers } from '@aiera/client-sdk/lib/hooks/useChangeHandlers';
 import { Message, useMessageListener } from '@aiera/client-sdk/lib/msg';
+import { Content } from '@aiera/client-sdk/modules/Content';
 import './styles.css';
 
-enum ContentType {
-    news,
-    corporateActivity,
+// TODO this should be generated by the server
+export enum ContentType {
+    news = 'news',
+    corporateActivity = 'corporateActivity',
 }
 
-interface ContentListSharedProps {
-    children?: ReactNode;
-}
-
-/** @notExported */
-export interface ContentListUIProps extends ContentListSharedProps {
-    company?: CompanyFilterResult;
-    onChangeSearch?: ChangeHandler<string>;
-    onSelectCompany?: ChangeHandler<CompanyFilterResult>;
-    onSelectContentType?: ChangeHandler<ContentType>;
-    searchTerm?: string;
-    selectedContentType?: ContentType;
-}
+export type ContentListContent = {
+    id: number;
+    date: string;
+    companyIdentifier: string;
+    exchangeName: string;
+    sourceName: string;
+    title: string;
+};
 
-const mockData = [
+export const mockBody: ReactElement = (
```

Comment:
Is this mocking the html from the server? If so I think it should be an HTML string instead of a react element just ot be sure we're properly rendering the HTML (since I think it will require dangerouslySetInnerHTML or something like that).